packages/yourloops/components/dashboard-widgets/medical-files/medical-report-list.tsx

```diff
@@ -29,42 +29,32 @@ import React, { type FunctionComponent, useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 
 import { makeStyles } from 'tss-react/mui'
-
-import DescriptionOutlinedIcon from '@mui/icons-material/DescriptionOutlined'
 import NoteAddIcon from '@mui/icons-material/NoteAdd'
 
 import Box from '@mui/material/Box'
 import Button from '@mui/material/Button'
-import IconButton from '@mui/material/IconButton'
 import List from '@mui/material/List'
-import ListItem from '@mui/material/ListItem'
-import ListItemIcon from '@mui/material/ListItemIcon'
-import ListItemText from '@mui/material/ListItemText'
-import Tooltip from '@mui/material/Tooltip'
-import Typography from '@mui/material/Typography'
 
 import { useAuth } from '../../../lib/auth'
 import MedicalFilesApi from '../../../lib/medical-files/medical-files.api'
 import MedicalReportEditDialog from '../../dialogs/medical-report-edit-dialog'
 import MedicalReportDeleteDialog from '../../dialogs/medical-report-delete-dialog'
-import TrashCanOutlined from '../../icons/trash-can-outlined'
 import { type CategoryProps } from './medical-files-widget'
 import { useAlert } from '../../utils/snackbar'
 import SpinningLoader from '../../loaders/spinning-loader'
 import {
   type MedicalReport,
-  type MedicalReportDialogPayload,
-  type MedicalReportWithIndex
+  type MedicalReportDeleteDialogPayload
 } from '../../../lib/medical-files/models/medical-report.model'
-import ListItemButton from '@mui/material/ListItemButton'
-import { getMedicalReportDate, getMedicalReportsToDisplay } from './medical-report-list.util'
+import { getSortedMedicalReports } from './medical-report-list.util'
+import { MedicalReportItem } from './medical-report-item'
 
 const useStyle = makeStyles()(() => ({
   categoryTitle: {
     fontWeight: 600
   },
   list: {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏If we could add `paddingTop: 0` here it would make the widget look much better ✨